### DISCUSSION TITLE
OG-Core Developer Meeting, Dec. 5, noon-1pm ET

### DISCUSSION THREAD

**Discussion by rickecon** [PSLmodels/OG-Core#829] [Meeting]
**Agenda for OG-Core developer meeting: 12/5/22**
12:00-1:00pm ET

Join Google Meet video conference
https://meet.google.com/zxz-fnrz-bkg

Join by phone
‪(US) +1 984-500-4439‬ PIN: ‪896 795 338‬#

This meeting is open to the public.

**Agenda**
* UN population data update for `demographics.py`

Next meeting: Monday, December 19, noon-1pm ET

@jdebacker @jpycroft @nikhilwoodruff

**Comment by jpycroft** [comment]
Hi Rick,

Apologies, I'll not be able to make it today.

I see you want to move forward with the UN population data.  I'll try to have a look in the coming days.

Best,

Jon


Get Outlook for Android<https://aka.ms/AAb9ysg>

________________________________
From: Richard Evans ***@***.***>
Sent: Monday, 5 December 2022, 06:55
To: PSLmodels/OG-Core ***@***.***>
Cc: jpycroft ***@***.***>; Mention ***@***.***>
Subject: [PSLmodels/OG-Core] OG-Core Developer Meeting, Dec. 5, noon-1pm ET (Discussion #829)


Agenda for OG-Core developer meeting: 12/5/22
12:00-1:00pm ET

Join Google Meet video conference
https://meet.google.com/zxz-fnrz-bkg<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fmeet.google.com%2Fzxz-fnrz-bkg&data=05%7C01%7C%7C8d6135e32a894849a9e108dad68dad94%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C638058201235156118%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=u5b6ZBmGmxb7wE%2BRXGfwM5gyMFYFld%2BZA6qCIRfGSv4%3D&reserved=0>

Join by phone
‪(US) +1 984-500-4439‬ PIN: ‪896 795 338‬#

This meeting is open to the public.

Agenda

  *   UN population data update for demographics.py

Next meeting: Monday, December 19, noon-1pm ET

@jdebacker<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fjdebacker&data=05%7C01%7C%7C8d6135e32a894849a9e108dad68dad94%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C638058201235312309%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=jkAht7c8vhkVrl8IVe6A6velMKpVBIqYhGVZcPkqxTU%3D&reserved=0> @jpycroft<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fjpycroft&data=05%7C01%7C%7C8d6135e32a894849a9e108dad68dad94%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C638058201235312309%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=HWg5FkPwdN4Wnnx%2FIe1z2J%2FL0J6Q3rfdwCekLntOR0c%3D&reserved=0> @nikhilwoodruff<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnikhilwoodruff&data=05%7C01%7C%7C8d6135e32a894849a9e108dad68dad94%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C638058201235312309%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=PBurFyEaJcQdJIXzqQGAVx6kSch%2BXNaCZomScqHPPGw%3D&reserved=0>
